optimize/util/optimize-commons/src/main/resources/service-config.yaml

```diff
@@ -335,6 +335,8 @@ import:
 # a connection to it.
 opensearch:
   connection:
+    # Use AWS credentials for authentication
+    awsEnabled: ${CAMUNDA_OPTIMIZE_OPENSEARCH_AWS_ENABLED:false}
```

Comment:
I am aware that the ENV var doesn't exactly match the path in the config file, but leaving the name like this to be consistent with Operate and Tasklist

Reply:
This makes sense to me 👍